lisa_commands/commands.py

```diff
@@ -510,6 +535,38 @@ def spotify_me(query, _slack_event):
     """
     return request.spotify_search(query)
 
+def song_me(query, _slack_event):
+    """
+    :param query: query str
+    :param _slack_event: a dict of slack event information(unused for this function)
+    :return: a link to spotify song item found by search
+    """
+    return request.spotify_query('track', query)
```

Comment:
It wasn't quite as straightforward as just redirecting it to `spotify_me`, as the original spotify me syntax was reliant on having the media type of the query within query itself, which it passed on to `request_helper` to be parsed. The new aliases don't have that, since the media type is now in the command itself, which doesn't get passed into `query`. 

I was able to work around it by moving the query type parsing inside of the command itself instead of request helper. It feels a little inelegant and I'm not really sure how much we want to split the command logic between files, but it's a little tough to support the original `spotify me x` syntax otherwise. Let me know if this works.